### PR TITLE
Fluentd prometheus

### PR DESCRIPTION
**Pull request by vwiessner**
Adds fluentd metrics to node_exporter output

### CODE REVIEW

File: runtime/opt/taupage/init.d/02-register-td-agent.py [hunk 1]
```diff
@@ -152,6 +152,21 @@ def update_configuration_from_template():
     if not logging_config:
         logger.info('Found no logging section in senza.yaml')
         raise SystemExit()
+    else:
+        try:
```
Comment (arjunrn) [inline runtime/opt/taupage/init.d/02-register-td-agent.py]: using a file context manager is a better pattern

Comment (vwiessner) [reply runtime/opt/taupage/init.d/02-register-td-agent.py]: good point, changed

**Comment by aermakov-zalando** [comment]
LGTM. Does it degrade gracefully if the log doesn't contain valid utf-8?

**Comment by vwiessner** [comment]
@aermakov-zalando can you go into detail what you mean? Fluentd doesn't crash, but the non UTF8 will show up garbled in Scalyr or on S3.

**Comment by aermakov-zalando** [comment]
That's fine, I just wanted to avoid a situation where Python would throw an encoding exception and not upload anything at all.

**Comment by vwiessner** [comment]
:+1:

**Comment by christianberg** [comment]
👍

**Pull request merged**